pyproject.toml

```diff
@@ -4,6 +4,7 @@
 # Defined by PEP 518
 requires = [
     "setuptools>=64",
+    "setuptools_scm>=7.0",
```

Comment:
In Iris we have here ` "setuptools_scm[toml]>=7.0",`

Not clear if that can help.
Just trying to understand the source of the carnage here !

Reply:
Another outside possibility is the lack of a 'lib' or 'src' directory.
That is not modern standard practice, perhaps it upsets scm ??
We have [this task](https://github.com/SciTools/iris-grib/issues/421) to shift the top-level folder , but I was put that off until after the sprint, as it will presumably break outstanding PRs.

Reply:
I don't think the `[toml]` thing should be a problem either, since @HGWright is already following the official guidance: [pypa/setuptools_scm: the blessed package to manage your versions by scm tags (github.com)](https://github.com/pypa/setuptools_scm)